src/core.js

```diff
@@ -82,6 +83,14 @@ migrateWarnProp( jQuery.expr, "filters", jQuery.expr.pseudos,
 migrateWarnProp( jQuery.expr, ":", jQuery.expr.pseudos,
 	"jQuery.expr[':'] is deprecated; use jQuery.expr.pseudos" );
 
+// Prior to jQuery 3.1.1 there were internal refs so we don't warn there
+if ( jQueryVersionSince( "3.1.1" ) ) {
+	migrateWarnFunc( jQuery, "trim", function( text ) {
+		return text == null ? "" : trim.call( text );
```

Comment:
The [3.x code](https://github.com/jquery/jquery/blob/3.4.1/src/core.js#L271) uses `(text + "")` to ensure the argument is stringified in case it's something unusual. Seems like we'd want to do that here too in order for any edge cases to work.

Reply:
Oh duh, we don't need it here!

Reply:
Hmm, maybe I should use the 3.x version to keep compat with Android 4.0... After all, people can use jQuery 3.x with Migrate 3.x to prepare for a future jQuery 4.x upgrade even if they’re not ready to drop browsers not supported by jQuery 4.x yet.

Reply:
Yeah, I agree with that. We have the code there and don't care about bytes in Migrate so we might as well use the same code.

Reply:
PR updated.